### ISSUE TITLE
Dependencies unchanged, error in new workspace

### ISSUE THREAD

**Issue by mwkaufman** [rearc/terraform-aws-codepipeline-status#3]
```
Error: error archiving directory: could not archive missing directory: .terraform/modules/pipeline_status_lambda/codepipeline-status-reporter-dependencies

  on .terraform/modules/pipeline_status_lambda/main.tf line 60, in data "archive_file" "dependencies_package":
  60: data "archive_file" "dependencies_package" {
```

This error happens when a Terraform apply occurs in one workspace and then the same code with an unchanged requirements.txt is applied in a new workspace. Since the pip install is not triggered, the zip of the output folder fails and the whole apply fails. This happens since we are Git ignoring the pip dependencies and triggering pip install based on changes to the requirements.txt.

**Comment by mwkaufman** [comment]
The current workaround is to run a terraform destroy in the new workspace and then run the apply to get out the new version. This introduces unnecessary downtime.

**Comment by mwkaufman** [comment]
Resolved in 0.1.3 release.
